### PR TITLE
[prometheus-node-exporter] Kubernetes recommanded labels node exporter

### PR DESCRIPTION
**Pull request by nlamirault**
#### What this PR does / why we need it:

Add Kubernetes recommanded labels : 

```yaml
---
# Source: prometheus-node-exporter/templates/serviceaccount.yaml
apiVersion: v1
kind: ServiceAccount
metadata:
  name: RELEASE-NAME-prometheus-node-exporter
  namespace: default
  labels:     
    app: prometheus-node-exporter
    app.kubernetes.io/component: prometheus-node-exporter
    app.kubernetes.io/instance: RELEASE-NAME
    app.kubernetes.io/managed-by: Helm
    app.kubernetes.io/name: prometheus-node-exporter
    app.kubernetes.io/version: "1.16.0"
    app.kubernetes.io/part-of: prometheus-node-exporter  
    chart: prometheus-node-exporter-1.16.0
    heritage: Helm
    release: RELEASE-NAME
  annotations:
    {}
imagePullSecrets:
  []
``` 

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Review by monotek — COMMENTED** [review]
What's the benefit of adding these labels if they are not used in the Matchlabels?

I would rather replace the old labels but as this would make this a breaking change the major version needs to be updated.

**Comment by nlamirault** [comment]
@monotek i add the labels to matchLabels et service selector.

**Comment by nlamirault** [comment]
@monotek any news ?

**Comment by nlamirault** [comment]
@monotek do you think we could reopen this PR ?

**Review by monotek — CHANGES_REQUESTED** [review]
Yes, but it needs changes.

This is not a bugfix rlease but a breaking change, which needs upgrading documentation.

Also please use the helm default for common and selector lables like "helm create" would add them.

File: charts/prometheus-node-exporter/README.md [hunk 1]
```diff
@@ -41,6 +41,10 @@ helm upgrade [RELEASE_NAME] [CHART] --install
 
 _See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documentation._
 
+### To 3.0.0
+
+￼Due to a change in DaemonSet labels, the upgrade requires `helm upgrade --force` in order to re-create the daemonset.
```
Comment (monotek) [inline charts/prometheus-node-exporter/README.md:46]: Thats not completlay true.
You also can do it like this: https://github.com/codecentric/helm-charts/blob/master/charts/keycloak/README.md#from-chart-versions--500

**Review by monotek — CHANGES_REQUESTED** [review]

**Pull request closed without merging**